### PR TITLE
Ukm

### PR DESCRIPTION
**Pull request by manggalang**

### CODE REVIEW

File: app/views/client/home/_ukm.html.erb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<div class="ukm">
+	<div class="ukm__title-wrap">
```
Comment (Fajaru1503) [inline app/views/client/home/_ukm.html.erb]: start class container-medium from here

Comment (Fajaru1503) [reply app/views/client/home/_ukm.html.erb]: remove title wrap, and add styling to ukm__title

File: app/views/client/home/_ukm.html.erb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		<h2 class="ukm__title">unit kegiatan mahasiswa stmik primakara</h2>
+	</div>
+	<div class="container-medium">
```
Comment (Fajaru1503) [inline app/views/client/home/_ukm.html.erb]: don't start container-medium from here

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 3]
```diff
@@ -0,0 +1,106 @@
+.ukm{
+	margin-top: 33px;
+
+	&__title-wrap{
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: remove title wrap, and add styling to the title instead

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+		flex-wrap: wrap;
+	}
+
+	&__item{
+		width: 305px;
+		box-sizing: border-box;
+		position: relative;
+		margin-top: 89px;
+
+		&:first-child{
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: make the nth-child(2) and nth-child(3) styling become one with this

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+		height: 299px;
+		background-size: cover;
+		transition: .3s opacity;
+	}
+
+	&__logo-primdev{
+		background-image: image-url("logo-gradient.png");
+		height: 80px;
+		width: 185px;
+		background-repeat: no-repeat;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: this useless

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+		position: absolute;
+		top: 50%;
+		left: 50%;
+		transform: translate(-50%, -50%);
+		background-size: contain;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: change it to cover

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+		filter: brightness(0) invert(1);
+		transition: all .3s ease-in-out;
+	}
+
+	&__content{
+		padding: 5px 0;
+		text-align: center;
+		border-top: 0;
+		transition: .3s ease-in-out;
+		position: relative;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: this is useless

File: app/views/client/home/_ukm.html.erb [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+		<div class="ukm__item-wrap">
+
+			<div class="ukm__item">
+				<a href="#" class="ukm__link">
+					<div class="ukm__banner-img"></div>
+					<div class="ukm__logo-primdev"></div>
+					<div class="ukm__content">
+						<h4 class="ukm__content-title">primakara developers</h4>
```
Comment (Fajaru1503) [inline app/views/client/home/_ukm.html.erb]: why you using h4? where is the h3?

File: app/views/client/home/_ukm.html.erb (hunk 1 above)
Comment (manggalang) [reply app/views/client/home/_ukm.html.erb]: okay, already to adding container-medium, and deleted wrap-title class

File: app/views/client/home/_ukm.html.erb (hunk 2 above)
Comment (manggalang) [reply app/views/client/home/_ukm.html.erb]: hmm sorry sir

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 3 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: already to deleted this class

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 4 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: okay, already combined nth-child 1 2 3 with nth-child(-n+3)

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 5 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: canceled

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 6 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: canceled

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 7 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: already to deleted this code

File: app/views/client/home/_ukm.html.erb (hunk 8 above)
Comment (manggalang) [reply app/views/client/home/_ukm.html.erb]: sorry sir,I'm not focused

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+	&__item-wrap{
+		display: flex;
+		justify-content: space-between;
+		flex-wrap: wrap;
+		margin-top: 55px;
+	}
+
+	&__item{
+		width: 305px;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: don't define it with pixel, use percentage

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+		box-sizing: border-box;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: it's useless

File: app/views/client/home/index.html.erb [hunk 11]
```diff
@@ -1 +1,2 @@
 <%= render 'client/home/headline' %>
+'<%= render 'client/home/ukm.html.erb' %>'
```
Comment (Fajaru1503) [inline app/views/client/home/index.html.erb]: remove the single quotation

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 9 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: okay, im already change this code with width 29.9%

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 10 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: okay

File: app/views/client/home/index.html.erb (hunk 11 above)
Comment (manggalang) [reply app/views/client/home/index.html.erb]: I forgot about that

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 12]
```diff
@@ -0,0 +1,158 @@
+.ukm{
+	margin-top: 33px;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: try to measure it with the ruler extension, recheck the gap from the mockup

File: app/assets/stylesheets/pages/home/_ukm.scss [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+		background-size: contain;
+		background-repeat: no-repeat;
+		filter: brightness(0) invert(1);
+		transition: all .3s ease-in-out;
+	}
+
+	&__content{
+		padding: 5px 0;
+		text-align: center;
+		border-top: 0;
```
Comment (Fajaru1503) [inline app/assets/stylesheets/pages/home/_ukm.scss]: it's useless

File: app/assets/stylesheets/pages/home/_ukm.scss (hunk 12 above)
Comment (manggalang) [reply app/assets/stylesheets/pages/home/_ukm.scss]: uh she up

**Pull request merged**